src/services/ObservationsService.ts

```diff
@@ -231,6 +236,18 @@ const exportGpx = async (observationId: number): Promise<any> => {
   }
 };
 
+const getSingleObservationResults = async (
+  observationId: number,
+  includeUnknown: boolean = false
+): Promise<Response2<ObservationResultsSingleResponsePayload>> => {
+  const response = await httpObservationResultsSingle.get2<ObservationResultsSingleResponsePayload>({
```

Comment:
I assume `get2` is what we should be using for new code. There's a mix of `get` and `get2` in this file but `get2` seems cleaner to me.